evaluators/text/BleuEvaluator/tests/test_bleuevaluator.py

```diff
@@ -0,0 +1,30 @@
+from .. import BleuEvaluator
+import pytest
+
+
+def test_bleu_evaluator():
+    desired = 'All cats are so super beautiful'
+    actual = 'All cats are so super beautiful'
+    assert BleuEvaluator().evaluate(actual, desired) == 1.0
+
+def test_bleu_evaluator_actual_emtpy():
+    desired = 'All cats are so super beautiful'
+    actual = ''
+    assert BleuEvaluator().evaluate(actual, desired) == 0.0
+
+def test_bleu_evaluator_desired_emtpy():
+    desired = ''
+    actual = 'All cats are so super beautiful'
+    assert BleuEvaluator().evaluate(actual, desired) == 0.0
+
+def test_bleu_evaluator_caps():
```

Comment:
Try to use pytest parametrization for these tests. Look at the other PRs on evaluators as inspiration

Reply:
Got it!